config/locales/fr.yml

```diff
@@ -0,0 +1,175 @@
+  feed_latest: "RubyGems.org | Derniers Gems"
+  time_ago: "il y a %{duration}"
+  form_disable_with: "Veuillez patienter..."
+  update: "Mise à jour"
+  edit: "Modification"
+  sign_in: "Connexion"
+  sign_up: "Inscription"
+  footer_about: "RubyGems.org est le service d&rsquo;hébergement de la communauté Ruby. Publiez vos gems instantanément et utilisez-les. Utilisez l'API pour interagir et trouver des informations sur les gems disponibles. Contribuez et améliorez ce site avec nous !"
+  locale_name: "Français"
+  this_rubygem_could_not_be_found: "Ce rubygem n'a pas pu être trouvé."
+
+  dashboards:
+    show:
+      latest: "Dernières mises à jour"
+      latest_title: "Flux RSS des dernières mises à jour"
+      no_subscriptions: "Vous ne vous êtes abonné à aucun email. Visitez %{gem_link} pour vous y abonner !"
+      gem_link_text: "la page d'un gem"
+      mine: "Mes Gems"
+      no_owned: "Vous n'avez pas encore publié de gem. Vous pouvez vérifier les guides de %{creating_link} ou de %{migrating_link} de gems depuis RubyForge."
+      creating_link_text: "création"
+      migrating_link_text: "migration"
+      my_subscriptions: "Abonnements"
+
+  home:
+    index:
+      downloads_counting: "téléchargements &amp; comptages"
+      find_blurb: "Trouvez, installez et publiez des RubyGems."
+      learn:
+        install_rubygems: "Installez RubyGems"
+
+  layouts:
+    application:
+      header:
+        dashboard: "Dashboard"
+        sign_in: "Connexion"
+        sign_up: "Inscription"
+        sign_out: "Déconnexion"
+        search_gem: "Recherche de Gems&hellip;"
+      footer:
+        status: "Statut"
+        uptime: "Uptime"
+        source_code: "Code"
+        data: "Données"
+        discussion_forum: "Discussions"
+        statistics: "Stats"
+        blog: "Blog"
+        about: "À propos"
+        help: "Aide"
+        api: "API"
+        guides: "Guides"
+        contribute: "Contribuer"
+        supported_by: "Soutenu par"
+        hosted_by: "Hébergé par"
+        designed_by: "Design par"
+        resolved_with: "Résolu par"
+        optimized_by: "Optimisé par"
+        tracking_by: "Tracking par"
+        monitored_by: "Monitoring par"
+        gems_served_by: "Gems mis à disposition par"
+
+  pages:
+    about:
+      title: "À propos"
+      purpose:
+        header: "Bienvenue sur %{site}, le service d'hébergement des gems de la communauté Ruby. Ce projet a trois objectifs :"
+        better_api: "Fournir une meilleure API pour gérer les gems"
+        transparent_pages: "Créer des pages de projet plus accessibles et transparentes"
+        enable_community: "Permettre à la communauté d'améliorer ce site"
+      founding: "Le projet a commencé en avril 2009 par %{founder}, et a grandi avec les contribution de %{contributors} et %{downloads}. À la sortie de RubyGems 1.3.6, ce site a été renommé de Gemcutter à %{title} from Gemcutter pour ancrer son rôle central dans la communauté Ruby."
```

Comment:
It's in the original text, I was just translating, so perhaps you should PR the EN content :)